packages/virtualized-lists/package.json

```diff
@@ -2,12 +2,16 @@
   "name": "@react-native/virtualized-lists",
   "version": "0.73.0",
   "description": "Virtualized lists for React Native.",
+  "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git@github.com:facebook/react-native.git",
+    "url": "https://github.com/facebook/react-native.git",
     "directory": "packages/virtualized-lists"
   },
-  "license": "MIT",
+  "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/virtualized-lists#readme",
```

Comment:
I am not fan of this deep linking based on HEAD as this will break if path or title changes.

Reply:
IIRC It is the correct way, for packages under monorepo. Though when path or title changes (rare case, as monorepo migration is completed), it'll break, may be the committers should take care of it in such scenarios.

About using HEAD based deep links, it **ensures to work** even in case we change the default branch or it's name. e.g. from `main` to `unstable`, etc.

Reply:
>  It is the correct way, for packages under monorepo 

Any examples? 

I see one problem documentation will diverge after published to npm by using HEAD. 
Also I noted that there is not readme for virtualized-lists, can you review all links better not add broken links.

Reply:
> > It is the correct way, for packages under monorepo
> 
> Any examples?

Currently all RN packages use HEAD based homepage links. Here:
https://github.com/search?q=repo%3Afacebook%2Freact-native+path%3Apackage.json+homepage&type=code

--
> I see one problem documentation will diverge after published to npm by using HEAD.

Afaik, something that might be not related but, since, currently packages are published manually and when it happens they are already ahead of main (in versions at least). So, using `main` based (generic) links would also cause same problems.

The optimal solution here is to change these urls during publish workflow, to include specific version based tree paths.
Which is not a difficult and shouldn't affect the current workflow.
Example: https://github.com/facebook/react-native/tree/0.72-stable/packages/virtualized-lists

--
> Also I noted that there is not readme for virtualized-lists, can you review all links better not add broken links.

Those are future proof urls, I know currently some packages which are published on npm doesn't have REDAMEs (when those are added we'll be already pointing to them), though, appending `#readme` (ie. fragment) **doesn't** break the urls.
- https://github.com/facebook/react-native/tree/HEAD/packages/virtualized-lists#readme - imho, is not a completely broken link

Reply:
We don't need to worry about README urls.

I'll add the missing READMEs in follow up PR whenever I get some free time from my academic work and ongoing exams 😃

Reply:
Appreciated, then it is better if you can create a new PR after adding Readme with link? Updating Node.js engine is a good change after all and like to merge that.

Reply:
Can you also comment on my latest comment below -> https://github.com/facebook/react-native/pull/37073#issuecomment-1521791948

Reply:
done.

Reply:
PR for adding missing READMEs and touchups to packages.json for all public RN packages: https://github.com/facebook/react-native/pull/37090